packages/core/content-manager/admin/src/hooks/useDocumentActions.ts

```diff
@@ -189,7 +192,7 @@ type IUseDocumentActs = ReturnType<UseDocumentActions>;
  *
  * @see {@link https://contributor.strapi.io/docs/core/content-manager/hooks/use-document-operations} for more information
  */
-const useDocumentActions: UseDocumentActions = () => {
+const useDocumentActions: UseDocumentActions = (fromPreview = false, fromRelationModal = false) => {
```

Comment:
Once my refactoring PR is merged we could avoid both of these by relying to context to infer them. Do you think this tracking PR could wait until then?

Reply:
is this hook only used in preview/relation modal?

Reply:
sure we can merge then. no need for todos i won't forget

Reply:
ok thanks